File: demo/src/main/java/com/example/android/entity/Region.java
package com.example.android.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("sys_region")
public class Region {
    @TableId(value = "_id", type = IdType.AUTO)
    private Long id;
    
    private String name;
    
    @TableField("region_id")
    private String code;
    
    @TableField("parent_id")
    private String parentId;
    
    private Integer level;
}

File: app/src/main/java/com/example/xiaoyi/model/Region.kt
package com.example.xiaoyi.model

data class Region(
    val id: Long,//地区ID
    val name: String,//地区名称
    val parentId: String,//父级地区编码
    val level: Int,//地区级别：1-省，2-市，3-区县，4-乡镇街道
    val code: String//地区编码
)
